test/unit-tests/test-output/_utils.ts

```diff
@@ -0,0 +1,5 @@
+export function getDummyStream(): any {
```

Comment:
Yeah, I sort of felt the same really it's so teensy, maybe would be nice to keep them all the same place but to be honest this is a nice to have if it's too much work we'll leave it I think because it's doing the job and unlikely to be needed elsewhere anyway :)